.github/workflows/vulnerability-scan.yml

```diff
@@ -4,6 +4,7 @@ on: pull_request
 permissions:
   pull-requests: write
   contents: write
+  issues: write
```

Comment:
I don't think this is related is it?

Reply:
I wasn't sure if the problem was caused by https://github.com/mshick/add-pr-comment/issues/43 or https://github.com/mshick/add-pr-comment/issues/67, so in the interest of speed I fixed both. I can remove this if we'd prefer to narrow this down now, or we can punt as this fixed seemed urgent?